### ISSUE TITLE
Embed tag is not working.

### ISSUE THREAD

**Issue by benjivm** [ExpressionEngine/ExpressionEngine#1718]
**Description of the problem**
The `{embed}` tag no longer works in 6.2.0, I see nothing in logs and debug mode shows nothing.

**How To Reproduce**
Use the embed tag.

**Error Messages**
Nothing in logs, no PHP errors, nada. PHP version does not affect this issue, tested on 7.4, 8.0, and 8.1.

Edit: in fact, no template tags appear to be working, only the raw HTML...

**Comment by TomJaeger** [comment]
Any chance you can enable debugging (set debug to 2 in your config) and see if there's an error?

**Comment by benjivm** [comment]
I have both debugging and developer log alerts on and see no errors: the Templates tab of the debug card seems fine to me and it even references the embeds that do not work:

```
    0.000008 / 8.8MB - Begin Template Processing -
    0.000014 / 8.8MB URI:
    0.000016 / 8.8MB Template: /
    0.000022 / 8.8MB Retrieving Template from Database: /index
    0.001904 / 8.9MB Retrieving Template from File
    0.014254 / 9.2MB Calling Extension Class/Method: Low_seg2cat_ext/template_fetch_template
    0.014959 / 9.3MB Calling Extension Class/Method: Stash_ext/template_fetch_template
    0.014984 / 9.3MB Template Type: webpage
    0.014989 / 9.3MB Parsing Site Variables
    0.016444 / 9.4MB Config Assignments & Template Partials
    0.016543 / 9.4MB Parse Current Time Variables
    0.016560 / 9.4MB Parsing Segment, Embed, Layout, logged_in_*, and Global Vars Conditionals
    0.020672 / 9.5MB - Beginning Tag Processing -
    0.020677 / 9.5MB Detecting Tags in Template
    0.020697 / 9.5MB No Closing Tag
    0.020740 / 9.5MB Running Tags
    0.021721 / 9.5MB Calling Tag: {exp:mixeer file='assets/img/massive-bg-3.jpg'}
    0.022150 / 9.5MB -> Data Returned
    0.025392 / 10.2MB Calling Tag: {exp:channel:entries channel="homepage" limit="1"}
    0.103302 / 19.1MB -> Data Returned
    0.103550 / 19.1MB Calling Tag: {exp:channel:entries channel="site-cards" category="23" limit="2"}
    0.130843 / 19.3MB -> Data Returned
    0.130884 / 19.3MB - End Tag Processing -
    0.131498 / 19.3MB - Processing Sub Templates (Depth: 1) -
    0.131505 / 19.3MB      List of Embeds: global/header,global/footer
    0.131518 / 19.3MB      Processing Sub Template: global/header
    0.131522 / 19.3MB      Retrieving Template from Database: global/header
    0.132814 / 19.3MB      Retrieving Template from File
    0.133634 / 19.3MB      Calling Extension Class/Method: Low_seg2cat_ext/template_fetch_template
    0.133823 / 19.3MB      Calling Extension Class/Method: Stash_ext/template_fetch_template
    0.133844 / 19.3MB      Template Type: webpage
    0.134198 / 19.3MB      Parsing Site Variables
    0.134232 / 19.3MB      Config Assignments & Template Partials
    0.134314 / 19.3MB      Parse Current Time Variables
    0.134334 / 19.4MB      Parsing Segment, Embed, Layout, logged_in_*, and Global Vars Conditionals
    0.138093 / 19.5MB       - Beginning Tag Processing -
    0.138099 / 19.5MB      Detecting Tags in Template
    0.138118 / 19.5MB      No Closing Tag
    0.138141 / 19.5MB      No Closing Tag
    0.138157 / 19.5MB      No Closing Tag
    0.138171 / 19.5MB      No Closing Tag
    0.138185 / 19.5MB      No Closing Tag
    0.138199 / 19.5MB      No Closing Tag
    0.138213 / 19.5MB      No Closing Tag
    0.138227 / 19.5MB      No Closing Tag
    0.138241 / 19.5MB      No Closing Tag
    0.138248 / 19.5MB      Running Tags
    0.138408 / 19.5MB      Calling Tag: {exp:mixeer file='favicon.ico'}
    0.138428 / 19.5MB       -> Data Returned
    0.138561 / 19.5MB      Calling Tag: {exp:mixeer file='apple-touch-icon-144x144.png'}
    0.138577 / 19.5MB       -> Data Returned
    0.138700 / 19.5MB      Calling Tag: {exp:mixeer file='apple-touch-icon-152x152.png'}
    0.138715 / 19.5MB       -> Data Returned
    0.138835 / 19.5MB      Calling Tag: {exp:mixeer file='favicon-32x32.png'}
    0.138850 / 19.5MB       -> Data Returned
    0.138970 / 19.5MB      Calling Tag: {exp:mixeer file='favicon-16x16.png'}
    0.138984 / 19.5MB       -> Data Returned
    0.139104 / 19.5MB      Calling Tag: {exp:mixeer file='assets/css/app.css'}
    0.139119 / 19.5MB       -> Data Returned
    0.139237 / 19.5MB      Calling Tag: {exp:mixeer file='assets/css/icons.css'}
    0.139251 / 19.5MB       -> Data Returned
    0.139368 / 19.5MB      Calling Tag: {exp:mixeer file='assets/js/app.js'}
    0.139383 / 19.5MB       -> Data Returned
    0.139504 / 19.5MB      Calling Tag: {exp:mixeer file='assets/img/kval-logo-lg.svg'}
    0.139519 / 19.5MB       -> Data Returned
    0.139525 / 19.5MB       - End Tag Processing -
    0.139528 / 19.5MB      Processing Advanced Conditionals
    0.141741 / 19.6MB      Calling Extension Class/Method: Stash_ext/template_post_parse
    0.141770 / 19.6MB      Processing Sub Template: global/footer
    0.141773 / 19.6MB      Retrieving Template from Database: global/footer
    0.142284 / 19.6MB      Retrieving Template from File
    0.143000 / 19.6MB      Calling Extension Class/Method: Low_seg2cat_ext/template_fetch_template
    0.143173 / 19.6MB      Calling Extension Class/Method: Stash_ext/template_fetch_template
    0.143190 / 19.6MB      Template Type: webpage
    0.143485 / 19.6MB      Parsing Site Variables
    0.143516 / 19.6MB      Config Assignments & Template Partials
    0.143585 / 19.6MB      Parse Current Time Variables
    0.143601 / 19.6MB      Parsing Segment, Embed, Layout, logged_in_*, and Global Vars Conditionals
    0.144069 / 19.6MB       - Beginning Tag Processing -
    0.144073 / 19.6MB      Detecting Tags in Template
    0.144088 / 19.6MB      No Closing Tag
    0.144096 / 19.6MB      Running Tags
    0.144231 / 19.6MB      Calling Tag: {exp:mixeer file='assets/js/newsletter.js'}
    0.144248 / 19.6MB       -> Data Returned
    0.144253 / 19.6MB       - End Tag Processing -
    0.144539 / 19.6MB      Calling Extension Class/Method: Stash_ext/template_post_parse
    0.144818 / 19.6MB Calling Extension Class/Method: Stash_ext/template_post_parse
    0.144831 / 19.6MB - End Template Processing -
    0.144832 / 19.6MB Parse Global Variables
    0.146877 / 19.6MB Template Parsing Finished
```

**Comment by TomJaeger** [comment]
Thanks for passing this along! 

Any chance you can try making this change?  To see if they then work as intended?

https://github.com/ExpressionEngine/ExpressionEngine/pull/1719/commits/fcbde011f2ec4b5226be5ff3af1411c49f6ea3fd

**Comment by benjivm** [comment]
Yes, replacing line 249 with the change in the link fixes the issue.

**Comment by TomJaeger** [comment]
Awesome, thanks for testing!

We're getting the 6.2.1 release ready currently, which will include this fix!

For anyone else following along, this is an issue when you have an extension using a template post parse hook.

**Comment by benjivm** [comment]
Which extension? Can you tell from the Template log above?

**Comment by TomJaeger** [comment]
Looks like Stash, hard to know 100% without disabling and re-enabling. 

While the extension in use is causing the error, it's really due to a change that was shipped in 6.2.0
